docs/reference/configuration.rst

```diff
@@ -287,6 +287,20 @@ values for OAuth2 client settings.
 See :github:`mozilla-services/cliquet-fxa`.
 
 
+Permission configuration
+::::::::::::::::::::::::
```

Comment:
That's why I only suggested to add ``# XXX`` in the code, since @ametaireau is working on permission docs. 
I don't think we have the glossary yet, no ?